website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

```diff
@@ -37,21 +37,10 @@ To enable Google Sign-in, go to the [Google API console](https://console.cloud.g
 ## Configure Appsmith instance  
 
 :::caution
-* The Client ID and Client Secret entered aren't verified by any means. Please test it out manually before disabling the Form login authentication method. If Form login is disabled before testing the Google authentication configuration setup, it may lead to the user getting stuck on the login screen if the Google configuration is invalid. You’ll have to update the environment variables manually to fix this, as you can't edit it via the Admin settings page since you won’t be logged in anymore.
-
+* Please manually test the Client ID and Client secret before disabling the Form login authentication. If the google configuration is invalid, the user is stuck on the login screen. To fix this, you’ll have to update the environment variables manually.
 * If you are running appsmith behind an ELB / Proxy, [configure Nginx](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```

Comment:
These two pieces of information do not tie up together. Maybe a rewrite could be:
```suggestion
* To fix signup issues, if you are running appsmith behind an ELB / Proxy, see [OAuth Sign up not working](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```
P.S. - can we get this confirmed by the technical team?